### ISSUE TITLE
Feature Request:  Merchant

### ISSUE THREAD

**Issue by LargePorg** [PlayersCommittee/gemp-swccg-public#563]
Godd afternoon, 
I had an idea for the merchant- to add in the ability to make your owned decipher cards into Vcards.
TL:DR below.

Preface:
There is the function of turning normal cards into foils but what about turning cards into Vcards?
As a returning player who has been focusing on the Decipher cards and slowly learning the Vcards, there is isn't a clear road from the one to the other.  
I did notice some posts on the forums showcasing which cards are needed for play etc but the merchant offers the unique opportunity to help new and returning players practice with the slips as if they were in hand-  cutting them out and sticking them on a decipher card.
I noticed in the gemp forum today that a new player asked what the purpose of the merchant was....   Johnny Chou replied: 
[Jun 1 12:58:29] Teacher: what is the "merchant" section used for? Is there a reason to "buy" cards?
[Jun 1 12:59:04] mryellow: No practical reason that I know of
Adding in the functionality of building Vcards from Decipher cards gives the merchant practical purpose- other than the virtual joy of ripping packs...   especially if it were free as it is in irl.

TL:DR/ Proposal:
So my proposal is:
1)  to add a button (similar to the buy/ sell buttons) to each Decipher card that has a Vcard associated with it: "View Vcard".   Clicking the button would display a popup window of the associated Vcard as if you shift Clicked it.
2) If the player also owns the Decipher card, add a second button: "Upgrade to Vcard".  Clicking the button would change the owned card to the Vcard version.
3) Adding a button to owned Vcards: "Downgrade card".  Clicking the button would change the owned Vcard to the Decipher version.
** I am willing to assist with the conception of this idea.  Please let me know.
Thanks, Thomas 
gemp: tjs625.   forums: tjs62500.  email: tjs62500@gmail.com.   discord: @tjs62500#7008.
